src/helpers/ToastHelper.ts

```diff
@@ -12,7 +12,7 @@ export enum ToastMessageType {
   DELETE,
   CHANGE_STATUS,
   ERROR,
-  CUSTOM_ERROR,
```

Comment:
It seemed more flexible to have a custom message in general, not just for errors. I had to define a custom message in my example anyway, because using existing ones would result in something like 'Login creation succeeded!' 😅